ctest.h

```diff
@@ -38,16 +38,23 @@ typedef void (*ctest_teardown_func)(void*);
     CTEST_IMPL_PRAGMA(GCC diagnostic ignored "-W" #w)
 #define CTEST_IMPL_DIAG_POP() \
     CTEST_IMPL_PRAGMA(GCC diagnostic pop)
+#ifndef __unused
```

Comment:
From C standard,
> All identifiers that begin with an underscore and either an uppercase letter or another underscore are always reserved for any use.

Which means you really shouldn't name the macro `__unused`. All such cases that were present in the past were changed to use the `CTEST_IMPL_` prefix instead, and this should be no exception.